tools/tags/tag-browser.css

```
@@ -6,8 +6,28 @@
   display: block;
 }
 
+.da-tag-browser {
+  position:relative;
+  overflow: hidden;
+  width: 100%;
+  height: 100vh;
+}
+
+.da-path-details {
+  height: 36px;
+  display: flex;
+  justify-content: space-between;
+  line-height: 36px;
+  font-size: 16px;
+  padding: 0 12px;
+}
+
 .da-tag-groups {
   display: flex;
+  overflow: auto;
+  position: absolute;
+  width:100%;
```

```suggestion
  width: 100%;
```
Did these not get caught by the linter?

---

They did not, we could add `declaration-colon-space-after`, although it looks like this rule is deprecated if we upgrade to Stylelint 15.